mpt/core/src/main/java/org/apache/james/mpt/protocol/ProtocolSession.java

```
@@ -520,6 +531,77 @@ public boolean isClient() {
         }
     }
 
+    protected enum TimerCommand {
+        START, PRINT, RESET;
+
+        public static TimerCommand from(String value) {
+            if (value.equalsIgnoreCase("start")) {
+                return START;
+            }
+            if (value.equalsIgnoreCase("print")) {
+                return PRINT;
+            }
+            if (value.equalsIgnoreCase("reset")) {
+                return RESET;
+            }
+            return null;
```

I would have throw a error instead of returning null